floor/test/integration/type_converter/order_database.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:async';
+
+import 'package:floor/floor.dart';
+import 'package:floor_annotation/floor_annotation.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sqflite/sqflite.dart' as sqflite;
+
+import 'order.dart';
+import 'order_dao.dart';
+import 'type_converter.dart';
```

Comment:
Point out that the type converter import can't be added automatically and thus has to be mentioned in the database definition.